### DISCUSSION TITLE
Can I download files to local folder

### DISCUSSION THREAD

**Discussion by TekCoder31** [dCache/nfs4j#118] [Q&A] [answered]
I'm looking for downloading the files from azure NetApp files to my local folder using NFS4.1. Is it possible using this library? Can someone help on this?

I can connect to the files successfully but could not stream the files as there is unsupported error thrown from a unimplemented method.

**Comment by paulmillar** [comment, marked as answer]
A similar question was asked in issue #97.  You may find the answers there helpful.
